index.html

```diff
@@ -0,0 +1,167 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>animePages</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/style.css">
+</head>
+
+<body>
+    <header class="header">
+        <section class="black-toolbar">
+            <ul class="blackT">
+                <li>
+                    <img src="ressources/icons/facebook.png" alt="">
+                </li>
+                <li>
+                    <img src="ressources/icons/twitter.png" alt="">
+                </li>
+                <li class="tes">English</li>
+                <li class="tes">My Page</li>
+                <li class="tes">Logout</li>
+            </ul>
+        </section>
+        <nav class="toolbar">
+            <a class="logo" href="#">
+                <img src="ressources/icons/logo.png" alt="">
+            </a>
+            <a class="button" href="#">
+                <img src="ressources/icons/mobile_menu 1.png" alt="Nav Button">
+            </a>
+            <ul class="cont-menu">
+                <ul class="menuDisplay">
+                    <li><a href="about.html" class="linkme" id="red-text" >About</a></li>
+                    <li><a href="index.html" class="linkme">Home</a></li>
+                    <li class="border"><a href="#">CC Campaign</a></li>
+                </ul>
+            </ul>
+        </nav>
+        <section class="overlay"></section>
+        <div class="menu-container">
+            <button type="button" class="close-button"><img src="/ressources/icons/Icon - Cancel.png"></button>
+            <ul class="menuDisplay">
+                <li>
+                    <a href="#section1">
+                        <img src="ressources/icons/logo.png" alt="">
+                    </a>
+                </li>
+                <li><a href="about.html" class="linkme" id="red-text1">About</a></li>
+                <li><a href="index.html" class="linkme">Home</a></li>
+                <li class="border"><a href="#">CC Campaign</a></li>
+            </ul>
+        </div>
+        <section class="front">
+            <div>
+                <h1>"Tomodachi, minogasanaide ne"</h1>
+                <h2>ATTACK ON TITAN<br>THE FINAL SEASON<br>PART 2</h2>
+                <p class="info-front">
+                    The Final Season (Part 2), which is scheduled to air in the fall of 2023!
+                    In the new PV, Armin recalls the memories of the three of them when they were young, and the line
+                    "Goodbye Ellen" of Mikasa who was also in the key image is symbolic!
+                </p>
+                <h3>Release date: October 2023</h3>
+                <p>Platform release: crunchyroll - wit studio NHK</p>
+            </div>
+        </section>
```

Comment:
- [x] Kindly make the **Headline section** outside of the` header` tag to follow the best practices of HTML and make the `main` tag for all of your section

Reply:
I'm sorry I don't understand the error very well